Assets/Scripts/с#/GetComponentEx6/Exercise5OffOn.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+public class Exercise5OffOn : MonoBehaviour
+{
+    private Switch _keyOn;
+
+    private void Start()
+    {
+        _keyOn = GetComponent<Switch>();
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Return))
+            _keyOn.enabled = !_keyOn.enabled;
+    }
+}
```

Comment:
What is this task?

Reply:
------------       I don't understand what needs to be done